_posts/devops/docker/2016-04-17-docker-installation.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: Docker - Installation
+author: Harshad_Yeola
+modified:
+comments: true
+categories: devops/docker
+excerpt:
```

Comment:
@harshadyeola when you open https://miteshshah.github.io/
Every post has small description what is post.

excerpt: "Step by step guide to install Docker on Ubuntu, Debian & CentOS"